{"name": "test_sample2", "status": "failed", "statusDetails": {"message": "AssertionError: assert 5 == 4", "trace": "def test_sample2():\n>       assert  5==4\nE       assert 5 == 4\n\nAPI_Python\\test_api_2.py:35: AssertionError"}, "start": 1702397331354, "stop": 1702397331354, "uuid": "d284a762-d29f-4d17-ad3a-bd3cb62278d5", "historyId": "878204a3f7dd8c036c65fb12fb1d6423", "testCaseId": "878204a3f7dd8c036c65fb12fb1d6423", "fullName": "API_Python.test_api_2#test_sample2", "labels": [{"name": "parentSuite", "value": "API_Python"}, {"name": "suite", "value": "test_api_2"}, {"name": "host", "value": "LAPTOP-CTE77PGL"}, {"name": "thread", "value": "10592-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "API_Python.test_api_2"}]}